lib/settings/widgets/accessibility_profile.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:dynamic_color/dynamic_color.dart';
+import 'package:flutter/material.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:thunder/core/enums/local_settings.dart';
+import 'package:thunder/core/singletons/preferences.dart';
+import 'package:thunder/settings/widgets/expandable_option.dart';
+import 'package:thunder/shared/snackbar.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+class AccessibilityProfile extends StatelessWidget {

```

Comment:
Thoughts on changing this from `AccessibilityProfile` to `SettingProfile` to make it more generic? It looks like this could be used across different profiles (including any potential custom user profiles if we support that in the future)

Reply:
Sure! Assuming you're just talking about the name of this widget and not the whole feature / setting. Done!